sources/models/scroll_function.js

```diff
@@ -1,8 +1,8 @@
 export function scrollToLastAddedElemen(component,id,show,select){
```

Comment:
the function is not necessary. it is used one and does selection and scrolling, so it is better to call these methods directly in the view. but you also need to wrap it into `waitSave` as it is safe to work with the real id from server-side than temporary

Reply:
not waitData (waits for loading), but waitSave (waits for saving)